usaspending_api/awards/models_matviews.py

```diff
@@ -152,6 +152,7 @@ class UniversalAwardView(models.Model):
     total_obligation = models.DecimalField(
         max_digits=15, decimal_places=2, blank=True,
         null=True)
+    description = models.TextField()
```

Comment:
This is referring to the award description?

Reply:
yes